src/main/java/mate/controller/manufacturer/DeleteManufacturerController.java

```diff
@@ -0,0 +1,25 @@
+package mate.controller.manufacturer;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/manufacturers/delete")
+public class DeleteManufacturerController extends HttpServlet {
+    private static final Injector injector =
+            Injector.getInstance("mate");
+
+    private final ManufacturerService manufacturerService =
+            (ManufacturerService) injector.getInstance(ManufacturerService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        manufacturerService.delete(Long.valueOf(req.getParameter("id")));
```

Comment:
After deleting a manufacturer, the user should be redirected to a specific page, usually the list of all manufacturers, to confirm the deletion.